lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -211,14 +212,57 @@ def add_prepare_artifacts_script_phase(native_target, spec)
             end
           end
 
+          # Adds a script phase that copies and strips dSYMs that are part of this target. Note this only deals with
+          # vendored dSYMs.
+          #
+          # @param [PBXNativeTarget] native_target
+          #         the native target for which to add the the copy dSYM files build phase.
+          #
+          # @return [void]
+          #
+          def add_copy_strip_dsyms_script_phase(native_target)
+            dsym_paths = target.framework_paths.values.flatten.reject { |fmwk_path| fmwk_path.dsym_path.nil? }.map(&:dsym_path)
+
+            if dsym_paths.empty?
+              UserProjectIntegrator::TargetIntegrator.remove_copy_resources_script_phase_from_target(native_target)
```

Comment:
is this correct? this looks like the normal `Copy Resources` script that's unrelated to dSYMs

Reply:
WHOOOPS good catch. Copy pasta.